core/domain/exp_jobs_one_off_test.py

```diff
@@ -419,6 +420,174 @@ def test_nonhuman_committers_not_counted(self):
+
+        # Run the job to compute contributors summary
+        job_id = exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.create_new() # pylint: disable=line-too-long
+        exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        exploration_summary = exp_services.get_exploration_summary_by_id(
+            exploration.id)
+
+        self.assertEqual(
+            3, exploration_summary.contributors_summary[user_a_id])
+
+    def test_contributors_with_only_reverts_not_included(self):
+        """Test that if only reverts are made by contributor then the
+        contributions summary shouldn’t contain that contributor’s ID
+        """
+
+        user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+        user_b_id = self.get_user_id_from_email(self.EMAIL_B)
+
+        # Let USER A make three commits.
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, user_a_id, title='Exploration Title 1')
+
+        exp_services.update_exploration(
+            user_a_id, self.EXP_ID, [{
+                'cmd': 'edit_exploration_property',
+                'property_name': 'title',
+                'new_value': 'New Exploration Title'
+            }], 'Changed title.')
+        exp_services.update_exploration(
+            user_a_id, self.EXP_ID, [{
+                'cmd': 'edit_exploration_property',
+                'property_name': 'objective',
+                'new_value': 'New Objective'
+            }], 'Changed Objective.')
+
+        # Let the second user revert version 3 to version 2
+        exp_services.revert_exploration(user_b_id, self.EXP_ID, 3, 2)
+
+        # Run the job to compute the contributors summary.
+        job_id = exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.create_new() # pylint: disable=line-too-long
+        exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        exploration_summary = exp_services.get_exploration_summary_by_id(
+            exploration.id)
+
+        # Check that the contributors_summary does not contains user_b_id
+        self.assertNotIn(user_b_id, exploration_summary.contributors_summary)
+
+        # Check that the User A has only 2 commits after user b has reverted
+        # to version 2
+        self.assertEquals(2, exploration_summary.contributors_summary[user_a_id]) # pylint: disable=line-too-long
+
+    def test_reverts_not_counted(self):
+        """Test that if both non-revert commits and revert are
+        made by contributor then the contributions summary shows
+        only non-revert commits for that contributor. However,
+        the commits made after the version to which we have reverted
+        shouldn't be counted either.
+        """
+
+        user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        # Let USER A make 2 non-revert commits and 1 revert
```

Comment:
Thanks for pointing this out @seanlip! I should have updated the comments as soon as I changed the code :-)